src/components/GlobeOptimized.jsx

```diff
@@ -518,8 +520,33 @@ export default function GlobeOptimized(props) {
       globeAttributes.current.is_clicked = 1;
       globeAttributes.current.meteorIndex = meteorPointIndex;
     } else {
+      const position = {
+        x:
+          globeAttributes.current.width -
+          globeAttributes.current.width / 2 -
+          (scaleFactor *
```

Comment:
Do we need to apply the scale factor to height parameter as well?

Reply:
Yes, it is required. I have tested without scaleFactor, not working properly.